pbrd/pbr_zebra.c

```diff
@@ -244,6 +244,11 @@ static int rule_notify_owner(ZAPI_CALLBACK_ARGS)
 		       __PRETTY_FUNCTION__, pbrms->installed);
 		break;
 	case ZAPI_RULE_FAIL_REMOVE:
+		/* Don't change state on rule removal failure */
+		DEBUGD(&pbr_dbg_zebra,
+		       "%s: Received RULE_FAIL_REMOVED: %" PRIu64,
```

Comment:
nit: `RULE_FAIL_REMOVE` (without the extra `D` at the end).

The four debug messages could potentially me merged in a single one if we write a function that maps `zapi_rule_notify_owner` enums to strings.